### DISCUSSION TITLE
Running on macOS via arm64 Apptainer image

### DISCUSSION THREAD

**Discussion by edp8489** [OpenRadioss/OpenRadioss#2125] [Show and tell]
I recently published a blog post describing the steps I took to set up a linux virtual machine and build the arm64 Apptainer image on my M2 Mac. Hopefully other Mac users out there find this useful!

[https://www.epeters.io/containerized-openradioss-macos](https://www.epeters.io/containerized-openradioss-macos)

**Comment by MarianBulla** [comment]
Hi Eric, 
yes, this is really great !!!
It enables the usage of OpenRadioss to the mac users.
Thanks for that 🥇 

Thanks and best regards,
Marian
